source/changelog.txt

```diff
@@ -12,6 +12,24 @@
    :depth: 1
    :class: singlecol
 
+v1.4.2
+------
+
+*Released May 18, 2022*
+
+- :issue:`MONGOSH-1139` - Adds Debian 11 support for ``mongosh``.
+
+- :issue:`MONGOSH-1183` - ``cursor.allowDiskUse()`` now accepts ``true``
+  or ``false``.
+
+- :issue:`MONGOSH-1204` - Adds visual identifier for Queryable 
```

Comment:
Is there a link to a Queryable Encryption page?

Reply:
I don't think we have anything to link out to yet.